### ISSUE TITLE
PDFJS Index size error for image PDFs

### ISSUE THREAD

**Issue by pkasibhatta** [mozilla/pdf.js#7825]
Link to PDF file (or attach file here):
[1239255-split-Operating+Statements YE 2015.pdf](https://github.com/mozilla/pdf.js/files/599339/1239255-split-Operating.Statements.YE.2015.pdf)

Configuration:
- Web browser and its version: IE 11.0.9600.18524
- Operating system and its version: Windows 7
- PDF.js version: 1.5.188
- Is an extension: No

Steps to reproduce the problem:
1.  Open Viewer and upload attached file
2. On some IE browsers a blank page is displayed and an error message - Index Size Error occurs when trying to call the drawImage method of the canvas.

What is the expected behavior? (add screenshot)
PDF should be rendered
What went wrong? (add screenshot)
Blank page is displayed
Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by Snuffleupagus** [comment]
Closing as duplicate of issue #6385.

**Comment by pkasibhatta** [comment]
Hi, 
I have checked the issue - 6385, only a workaround has been given - 
Work-around: Disable HWA (IE: Internet Options -> Advanced -> Use software rendering instead of GPU rendering).

Is there any other solution which does not require setting change from user side?

**Comment by Snuffleupagus** [comment]
> I have checked the issue - 6385, only a workaround has been given -
> Work-around: Disable HWA (IE: Internet Options -> Advanced -> Use software rendering instead of GPU rendering).
> 
> Is there any other solution which does not require setting change from user side?

Please note that closing an issue as "duplicate" doesn't necessarily imply that it's solved; closing duplicates is simply done to keep the number of open issues more manageable (and is commonly done in all bug trackers).

Solving this issue is blocked on e.g. #6418 and #6419 (at least one of which are linked from issue #6385).
